app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -265,6 +248,18 @@ class ThreadFragment : Fragment() {
         phoneContextualMenuAlertDialog.initValues(mainViewModel.snackBarManager)
     }
 
+    private fun observeThreadOpening() = with(binding) {
+
+        mainViewModel.currentThreadUid.distinctUntilChanged().observeNotNull(viewLifecycleOwner) { threadUid ->
```

Comment:
We can use only one LiveData, but the logic will still be split because it won't be observed in the same Fragment.
Here, it's the ThreadFragment, but the closing of the Thread is in the ThePaneFragment.

Reply:
Done !